### PR TITLE
Double click on the dependency graph navigates to the mod

### LINKED ISSUE #1082: Enhancement: Add the ability to navigate to a mod from the "Relationships" tab in the gui client
Saw this on the [KSP forums](http://forum.kerbalspaceprogram.com/threads/100067-The-Comprehensive-Kerbal-Archive-Network-%28CKAN%29-Package-Manager-v1-6-21-5-Jun-2015?p=2001947&viewfull=1#post2001947) posted by deeperman and thought it was a good idea:

"please add the ability to navigate to the desired modification"
http://imgur.com/h7olv5e

Essentially, when a user double clicks on a mod in the relationship viewer it changes the selected mod (or changes the filtering) to that mod.

### PR DESCRIPTION
**Pull request by Postremus**
closes #1082

### CODE REVIEW

File: GUI/MainModInfo.cs [hunk 1]
```diff
@@ -92,6 +92,7 @@ private TreeNode UpdateModDependencyGraphRecursively(TreeNode parentNode, Module
             }
 
             var node = parentNode == null ? new TreeNode(nodeText) : parentNode.Nodes.Add(nodeText);
+            node.Name = module.name;
```
Comment (pjf) [inline GUI/MainModInfo.cs]: Everything looks good, but just checking this change here... Without it I presume the dependency graph could continue to show the old mod name in the case that we double-clicked on dependency mod to reach it?

Comment (Postremus) [inline GUI/MainModInfo.cs]: Without the node.Name property I wouldn't know which mod to show on double click. It has nothing to do with refreshing the dependency graph.

Comment (pjf) [inline GUI/MainModInfo.cs]: Awesome, thank you again! :)

**Comment by pjf** [comment]
Merged with thanks. :)

**Pull request merged**